numpy/core/src/umath/_umath_tests.c.src

```diff
@@ -598,25 +597,15 @@ static struct PyModuleDef moduledef = {
         NULL,
         NULL
 };
-#endif
 
 /* Initialization function for the module */
-#if defined(NPY_PY3K)
 #define RETVAL(x) x
```

Comment:
`RETVAL` is now a pointless alias. Depending on how much should be done in one go, could potentially remove it as well.

Reply:
I'm planning to clean it up in a 2nd pass.